StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1028UnitTests.cs

```diff
@@ -8,6 +8,7 @@
     using StyleCop.Analyzers.SpacingRules;
     using TestHelper;
     using Xunit;
+    using System.Text;
```

Comment:
Oh great, but I feel a little bit ashamed. In my VS2013 environment it's automatically fixed when I save a file and StyleCop warns me about it.